tests/driver_sds011/main.c

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (C) 2018 HAW-Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the SDS011 Laser Dust Sensor driver
+ *
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ *
+ * @}
+ */
+#include <string.h>
+#include <stdio.h>
+#include <stdint.h>
+
+#include "xtimer.h"
+#include "color.h"
+
+#include "sds011.h"
+#include "sds011_params.h"
+#include "msg.h"
+#include "thread.h"
+
+#define ACTIVE_REPORTING_TEST_DURATION_S         (30U)
+#define PUT_TO_QUERY_MODE_RETRIES                (3)
+#define PUT_TO_QUERY_MODE_RETRY_TIMEOUT_MS       (100)
+#define MANUAL_QUERY_CNT                         (10)
+#define WORKING_PERIOD                           (0)
+
+char print_thread_stack[THREAD_STACKSIZE_MAIN];
+
+/**
+ * @brief   Allocate the device descriptor
+ */
+static sds011_t dev;
+
+static char* _rmode_str(sds011_reporting_mode_t rmode) {
```

Comment:
this file still does not adhere to RIOT coding style, e.g. brace should be on next line

Reply:
damn! fixed it - may i squash?